src/c-writer.cc

```diff
@@ -2360,7 +2361,7 @@ void CWriter::PushFuncSection(std::string_view include_condition) {
 void CWriter::Write(const Func& func) {
   func_ = &func;
   // Copy symbols from global symbol table so we don't shadow them.
```

Comment:
I guess this comment is not longer accurate?

I guess its not possible for a global symbol could start with `var_`?

Reply:
Global symbols should either start with `w2c_` or `wasm2c_`, and therefore local symbols would not shadow global symbols. Deleted this comment in [ce882d6](https://github.com/WebAssembly/wabt/commit/ce882d6a23803390ab098d908c1df2bac501651a).